config/locales/es.yml

```diff
@@ -2959,6 +2959,15 @@ es:
           helper_heading: El lugar donde vivo no está en la lista
           municipality: Municipio
           title: Seleccione el municipio donde residía el 31 de diciembre de %{filing_year}
+      nj_primary_state_id:
```

Comment:
[dust] What are New Jersey's practices for translation files? We usually will run our English copy through google translate to get Spanish copy as a stand-in before transfix translators update it (which will include NJ strings) but that might not happen for awhile so good to have a stand-in. I see that some NJ copy has been translated and some has not. Is this an agreed upon practice?

Reply:
good question - it's our NJ team's practice to just drop in English for now and know that it will be translated later. i asked the same question [here on an earlier PR](https://github.com/codeforamerica/vita-min/pull/4951#discussion_r1833034706)